HopsanGenerator/templates/fmu1CsImportComponentTemplate.hpp

```diff
@@ -180,6 +180,14 @@ class <<<className>>> : public <<<classParent>>>
             return;
         }
 
+        //Set parameters
+        fmi1_value_reference_t vr;
+        double value;
+>>>setpars>>>        vr = <<<vr>>>;
```

Comment:
This only work for real type parameters, text will cause compilation failure, and int/bool would just be cast to double

Reply:
hmm apparently I could push directly to your branch :)